alacritty_terminal/src/config/mod.rs

```diff
@@ -149,6 +161,7 @@ impl Default for Cursor {
             vi_mode_style: Default::default(),
             thickness: Percentage::new(DEFAULT_CURSOR_THICKNESS),
             unfocused_hollow: Default::default(),
+            blink_rate: 666,
```

Comment:
I personally find neither 750 nor 500 ideal. So I picked the next best thing in between. Though I might have just been biased based on the other terminals I tested.

Reply:
I think it's nice, since I don't even like the default you've picked, and I'd image that it'll be hard to pick a reasonable one, since you like 666, but I don't find it good. Also, even gtk allows changing it.

Reply:
Fair enough. Which would be your preferred blink rate? I don't really like blinking to begin with.